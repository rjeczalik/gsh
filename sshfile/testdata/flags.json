[
	["-o", "StrictHostKeyChecking=no", "-o", "GlobalKnownHostsFile=/dev/null", "-o", "UserKnownHostsFile=/dev/null"],
	["-o", "StrictHostKeyChecking=no", "-o", "GlobalKnownHostsFile=/dev/null", "-o", "UserKnownHostsFile=/dev/null", "-o", "TcpKeepAlive=yes", "-o", "ConnectTimeout=10"],
	["-o", "StrictHostKeyChecking=no", "-o", "GlobalKnownHostsFile=/dev/null", "-o", "UserKnownHostsFile=/dev/null", "-o", "TcpKeepAlive=yes", "-o", "ConnectTimeout=10"],
	["-o", "StrictHostKeyChecking=no", "-o", "GlobalKnownHostsFile=/dev/null", "-o", "UserKnownHostsFile=/dev/null", "-o", "TcpKeepAlive=yes", "-o", "ConnectTimeout=10", "-o", "ConnectionAttempts=3", "-o", "ServerAliveInterval=60", "-o", "ServerAliveCountMax=5"]
]
