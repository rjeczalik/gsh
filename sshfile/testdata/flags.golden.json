[
	{
		"stricthostkeychecking": "no",
		"globalknownhostsfile": "/dev/null",
		"userknownhostsfile": "/dev/null",
		"host": ""
	},
	{
		"stricthostkeychecking": "no",
		"globalknownhostsfile": "/dev/null",
		"userknownhostsfile": "/dev/null",
		"tcpkeepalive": "yes",
		"connecttimeout": "10",
		"host": ""
	},
	{
		"stricthostkeychecking": "no",
		"globalknownhostsfile": "/dev/null",
		"userknownhostsfile": "/dev/null",
		"tcpkeepalive": "yes",
		"connecttimeout": "10",
		"host": ""
	},
	{
		"stricthostkeychecking": "no",
		"globalknownhostsfile": "/dev/null",
		"userknownhostsfile": "/dev/null",
		"tcpkeepalive": "yes",
		"connecttimeout": "10",
		"connectionattempts": "3",
		"serveraliveinterval": "60",
		"serveralivecountmax": "5",
		"host": ""
	}
]